### ISSUE TITLE
[FE] Severity 8: Web #14 - When the user activates "Privacy Act Statement" control and the modal dialog appears,  visual focus is set to the "Close Privacy Act Statement modal" button,  while the screen reader reads out the entire contents of the dialog at once.  The consequence is that all text runs together,  and the SR does not announce semantic elements,  such as headings.   

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#59327]
## Notes
LP: I'm guessing that we don't have any control over this control?

## Issue
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/86678742/11427c59-a3db-4549-b20a-5353ede175d9)

Original Spreadsheet Source for Issue: [Mobile Audit 05102023 Patient Check-in (eCheck-in  Pre-Check-in) Web.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/11614848/Mobile.Audit.05102023.Patient.Check-in.eCheck-in.Pre-Check-in.Web.xlsx)

## Tasks
- [ ] [FE] Resolve issue
- [ ] Create or update unit tests
- [ ] Create or update e2e tests
- [ ] Update relevant documentations (readme and use cases)

## Acceptance Criteria
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Tested on staging
- [ ] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature
- [ ] Prod validation:
  - [ ] not necessary
  - [ ] scheduled

## Code Review Steps
- [ ] Read ticket details and verify that ACs are met
- [ ] Review code in PR
- [ ] Run branch locally
- [ ] Check that new and updated translations use new keys
- [ ] Check for hardcoded strings

**Comment by lee-delarm6** [comment]
I believe this is a Design team issue, it launches using the `VaModel` component

**Comment by loripusey** [comment]
Submitted Platform support ticket for the Design System [here](https://dsva.slack.com/archives/CBU0KDSB1/p1687293774975819); closing
